artifacts/manifests/006_service.yaml

```diff
@@ -0,0 +1,15 @@
+apiVersion: v1
+kind: Service
+metadata:
+  name: clusterresourceoverride
+  namespace: cluster-resource-override
+  labels:
+    clusterresourceoverride: "true"
+  annotations:
+    service.alpha.openshift.io/serving-cert-secret-name: server-serving-cert
```

Comment:
This creates `server-serving-cert` secret, which it seems you are creating manually (e.g. `004_secret.yaml` and `hack/load-cert-into-manifest.sh`)

Reply:
It's a copy paste error, I removed it.